Gemfile

```diff
@@ -18,7 +18,7 @@ gem 'mongoid', '~> 7.0'
 gem 'oai'
 gem 'ougai', '~> 1.8'
 gem 'parse-cron'
-gem 'puma', '~> 3.12'
+gem 'puma'
```

Comment:
that's for supporting the `raise_exception_on_sigterm false` that was deployed in the previous release